zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowModel.java

```diff
@@ -182,12 +187,13 @@ public static Builder builder() {
     }
   }
 
-  private static MetaGraphDef extractMetaGraphDefinition(final SavedModelBundle bundle) {
+  private static MetaGraphDef extractMetaGraphDefinition(final SavedModelBundle bundle)
+      throws TensorflowMetagraphParsingException {
     final MetaGraphDef metaGraphDef;
     try {
       metaGraphDef = MetaGraphDef.parseFrom(bundle.metaGraphDef());
     } catch (InvalidProtocolBufferException e) {
-      throw new RuntimeException("Failed parsing tensorflow metagraph definition", e);
+      throw new TensorflowMetagraphParsingException(e);
```

Comment:
@guidj what about setting the msg here?

Reply:
I can do that. Not sure what the better practice is here. I guess if info is provided around where the issue occurred or why if possible that'd be better